app/views/form-designer/pages/_routes.js

```diff
@@ -0,0 +1,823 @@
+const govukPrototypeKit = require('govuk-prototype-kit')
+const router = govukPrototypeKit.requests.setupRouter()
```

Comment:
```suggestion
const router = govukPrototypeKit.requests.setupRouter('/pages')
```

Reply:
Thanks for taking a look @lfdebrux. I’ve tried altering the route files, and no matter what way I do it if I don’t include all the lines as they are it doesn’t seem to work.

Reply:
Okay, ignore my suggestions then. I'm not sure why it's not working, but I won't mess with what you've got!